### PR TITLE
Internal Updates

### PR DESCRIPTION
**Pull request by nakirekommula**
### Description
Adds AdListener support for NativeAd.

***
### Testing
N/A

***
### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [x] New feature. A non-breaking change which adds functionality.
- [ ] Other, such as a build process or documentation change.
***
#### Notes
Internal Updates only.

### CODE REVIEW

**Comment by jonsimantov** [comment]
Will any of this break with the iOS 11.0.1 release? (I'm in the process of updating to it now.)

**Comment by nakirekommula** [comment]
> Will any of this break with the iOS 11.0.1 release? (I'm in the process of updating to it now.)

Doesn't look like workflows related to adlistener are changed in the recent release. So this update should be compatible.

File: gma/src/android/gma_android.cc [hunk 1]
```diff
@@ -973,6 +973,31 @@ void JNI_NativeAd_completeLoadedAd(JNIEnv* env, jclass clazz, jlong data_ptr,
   env->DeleteLocalRef(j_response_info);
 }
 
+void JNI_NativeAd_notifyAdClicked(JNIEnv* env, jclass clazz, jlong data_ptr) {
+  firebase::gma::internal::NativeAdInternal* internal =
```
Comment (jonsimantov) [inline gma/src/android/gma_android.cc:977]: Should add some asserts here like the other JNI native functions.

**Review by jonsimantov — APPROVED** [review]
Approved with one small issue

**Comment by nakirekommula** [comment]
> Approved with one small issue

Addressed the comment. Thank you for the quick review Jon!

**Pull request merged**